src/calculation/HTMLBuilder.ts

```diff
@@ -92,23 +92,49 @@ Handlebars.registerHelper('highlightCoverage', (localId, context) => {
  */
 export function sortStatements(measure: fhir4.Measure, groupId: string, statements: StatementResult[]) {
   const group = measure.group?.find(g => g.id === groupId) || measure.group?.[0];
-  const populationSet = new Set(group?.population?.map(p => p.criteria.expression));
+  const populationOrder = [
+    'initial-population',
+    'denominator',
+    'denominator-exclusion',
+    'denominator-exception',
+    'numerator',
+    'numerator-exclusion',
+    'measure-population',
+    'measure-population-exclusion',
+    'measure-observation'
+  ];
+
+  const populationIdentifiers: Record<string, string> = {};
```

Comment:
Quick comment saying that this is a lookup of cql expression identifier -> population type might be useful